src/API/fetchingAPI.js

```diff
@@ -0,0 +1,23 @@
+const API_KEY = 'c1c49f469a4a562fcad4008d7e66762d';
+const BASE_URL = 'https://financialmodelingprep.com/api/v3';
+
+export const fetchMetricesAPI = async () => {
+  // eslint-disable-next-line no-console
+  fetch(`${BASE_URL}/stock_market/actives?apikey=${API_KEY}`).then((res) => res.JSON()).then((data) => console.log(data));
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃